eng/containers/ci.yml

```diff
@@ -0,0 +1,39 @@
+pr: none
+
+trigger:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - eng/containers/
+
+pool:
+  vmImage: 'ubuntu-20.04'
```

Comment:
Might want to use the 1ES pool

Reply:
Good catch, fixed (copied from the now inactive net definition).